pmd-cataloger/build.gradle.kts

```diff
@@ -78,3 +79,20 @@ tasks.named("assemble") {
   dependsOn("installPmd")
 }
 
+tasks.test {
+  finalizedBy(tasks.jacocoTestReport) // Report is always generated after test runs.
+}
+
+tasks.jacocoTestReport {
+  dependsOn(tasks.test)
+}
+
+tasks.jacocoTestCoverageVerification {
+  violationRules {
+    rule {
+      limit {
+        minimum = "0.80".toBigDecimal()
```

Comment:
I'm enforcing a minimum of 80% instead of 90%, because the coverage is exactly 80% at this time.